controllers/core/openstackversion_controller.go

```diff
@@ -253,7 +263,17 @@ func (r *OpenStackVersionReconciler) Reconcile(ctx context.Context, req ctrl.Req
 			Log.Info("Minor update for Controlplane in progress")
 			return ctrl.Result{}, nil
 		}
+
 		// TODO minor update for Dataplane in progress goes here
+		if !openstack.DataplaneNodesetsDeployedCheck(instance, dataplaneNodesets) {
```

Comment:
ditto I would drop the Check from end of the function name

Reply:
done